docs/sources/send-data/promtail/stages/non_indexed_labels.md

```diff
@@ -5,12 +5,16 @@ description: non_indexed_labels stage
 # non_indexed_labels
 
 {{% admonition type="warning" %}}
-Non-indexed labels is an experimental feature and is subject to change in future releases of Grafana Loki.
+Non-indexed labels is an experimental feature and is subject to change in future releases of Grafana Loki. 
```

Comment:
please pull the main, I have renamed this file

Reply:
Done